mantidproject/mantid
# Moving TF asymmetry box

The muon group wanted the TF asymmetry tick box to move into the data analysis tab (from the settings tab). 

**To test:**
1. Load some data into muon analysis, 
2. Go to the settings tab and make sure that multiple fitting is not selected.
3. Go to the data settings tab
4. If the TF asymmetry box is ticked (in the settings table) then you should have normalization and fix normalization in the data table. In the fit menu should be TF Asymmetry fit. If the TF asymmetry box is unticked these three things should not be present. 
5. Go to the settings tab and turn on multiple fitting.
6. In the data analysis tab the normalization and TF Asymmetry fit should be present if and only if the TF asymmetry box is ticked


Fixes #19987. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
